### DISCUSSION TITLE
Pass-through processing disabled while movie_passthrough is on

### DISCUSSION THREAD

**Discussion by ceewanna** [Motion-Project/motionplus#32] [General]
I got this warning though movie_passthrough is on. My camera netcam_url and netcam_high_url are of rtsp type where netcam_params allowed_media_types=video. 

The netcam_high_url comes with both video and audio but I haven't set anything there yet. What should be set for netcam_high_params? Can I include things like preset=ultrafast,crf=32,fps_mode=vfr ? (which are ffmpeg settings)

**Comment by Mr-Dave** [comment]
First, the `Pass-through processing disabled` reason varies so seeing the precise messages from the log should provide more context of whether it is the camera or version of ffmpeg or something else.

Regarding the additional options for ffmpeg, you are correct that in how they would be specified.  However, for a pass through source, they do not matter.  The ones listed are associated with encoding and pass through does no encoding.  It takes a packet from the source and writes the packet out.

**Reply by ceewanna** [reply]
This is the recent log file I have.
[motionplus.log](https://github.com/Motion-Project/motionplus/files/10662108/motionplus.log)

**Comment by Mr-Dave** [comment]
Thanks.  There is a bug.  I just pushed the fix to the master branch.

**Comment by ceewanna** [comment]
Oh there's another issue which I am not so sure related to this.
It seems there is this: netcam_connect: High: An audio stream was detected.  Capture_rate increased to compensate.
But audio doesn't show up in the movie output.

I just re-built the whole system and it seemed to work fine now with audio in stream too. Thanks so much.
